database/migrations/2019_04_05_180853_change_private_reference_number_column_type.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class ChangePrivateReferenceNumberColumnType extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('attendees', function ($t) {
```

Comment:
A small nitpick but please add the type indicator to the closure of both `up()` and `down()` methods? 

```
function (Blueprint $table) {
```

and use the full variable name for the migration in the `up()` and `down()` calls please? 🙏